packages/domain-picker/src/utils/index.ts

```diff
@@ -54,11 +57,15 @@ type DomainSuggestionsVendor =
 	| 'link-in-bio'
 	| 'link-in-bio-tld'
 	| 'newsletter'
-	| 'ecommerce';
+	| 'ecommerce'
+	| 'gravatar';
 
 export function getDomainSuggestionsVendor(
 	options: DomainSuggestionsVendorOptions = {}
 ): DomainSuggestionsVendor {
+	if ( isDomainForGravatarFlow( options.flowName ) ) {
+		return 'gravatar';
```

Comment:
You can change this for `variation2_front` while the problem with `.link` domains in the test environment is not fixed